src/AppInstallerCommonCore/Public/winget/ManifestValidation.h

```diff
@@ -48,6 +48,9 @@ namespace AppInstaller::Manifest
         const char* const MissingManifestDependenciesNode = "Dependency not found: ";
         const char* const NoSuitableMinVersion = "No Suitable Minimum Version: ";
         const char* const FoundLoop = "Loop found.";
+        const char* const ExceededAppsAndFeaturesEntryLimit = "Only zero or one entry for Apps and Features can be specified for InstallerType portable.";
+        const char* const ExceededCommandsLimit = "Only zero or one value for Commands can be specified for InstallerType portable.";

```

Comment:
```suggestion
        const char* const ExceededAppsAndFeaturesEntryLimit = "Only zero or one entry for Apps and Features may be specified for InstallerType portable.";
        const char* const ExceededCommandsLimit = "Only zero or one value for Commands may be specified for InstallerType portable.";
```
nit: grammar for clarity

Reply:
Fixed grammar